tasks/src/automatic-speech-recognition/about.md

```diff
@@ -1,3 +1,23 @@
+## Use Cases
+
+### Virtual Speech Assistants
+
+Many end devices have embedded a virtual assistance as a better way to interact with end users. These assistances rely on ASR models to recognize different voice commands in order to perform specified tasks. For instance, you might ask your phone for dialing a phone number, asking for an answer or scheduling a meeting. 
+
+### Caption Generation
+
+A Caption Generation model can take the audio from sources such as live streamed videos or recorded videos to generate automatic captions. This can help with content accessibility for example: audiences who are watching in a non-native language can rely on captions to interpret the content. Furthermore, it helps with information retention.
+
+## Task Variants
+
+### Multilingual ASR
+
+These models are intended to transcript multiple languages from a given audio. Some models might incorporate [language identification](https://huggingface.co/tasks/audio-classification) blocks to improve multilingual ASR. 
```

Comment:
```suggestion
Multilingual ASR models can convert audio inputs with multiple languages into transcripts. Some multilingual ASR models include [language identification](https://huggingface.co/tasks/audio-classification) blocks to improve the performance.
```
We try to avoid passive voice as it makes it harder to read 🙂